torch/_dynamo/variables/constant.py

```diff
@@ -27,6 +27,10 @@
 
 
 class ConstantVariable(VariableTracker):
+    @staticmethod
+    def create(value, **kwargs):
+        return ConstantVariable(value, **kwargs)
```

Comment:
How are we going to prevent people from calling the constructor directly?

Reply:
We load it up with useful asserts that instruct users to do the right thing. Stacked on this Pr is the first steps of that.

If there's a better way, lmk, but I dont think we get to mark it private in python in any meaningful way.